### PR TITLE
Grant Reighard

### PR DESCRIPTION
**Pull request by grantreighard**

### CODE REVIEW

File: basic-javascript/project-1.spec.js [hunk 1]
```diff
@@ -1,3 +1,150 @@
...
+    expect(helpers.getCircleArea(5)).toBeCloseTo(78.54);
+    expect(helpers.getCircleArea(100)).toBeCloseTo(31415.93);
+    expect(helpers.getCircleArea(20)).toBeCloseTo(1256.637);
+});
+
+it('should return length times width times height', () => {
+    expect(helpers.getRectangularPrismVolume(10, 10, 10)).toBe(1000);
+    expect(helpers.getRectangularPrismVolume(5, 10, 25)).toBe(1250);
+    expect(helpers.getRectangularPrismVolume(3, 4, 5)).toBe(60);
+});
```
Comment (adfaris) [inline basic-javascript/project-1.spec.js]: Your test looks good but you should think about testing edge cases, like when a user put string instead of a number. you test should handle those.

File: basic-javascript/project-2.spec.js [hunk 2]
```diff
@@ -1,3 +1,111 @@
...
+    expect(funcs.averageTestScore([1, 2, 3, 4, 5])).toBe(3);
+    expect(funcs.averageTestScore([10, 20, 30, 40, 50])).toBe(30);
+    expect(funcs.averageTestScore([111, 222, 333, 444, 555])).toBe(333);
+});
+
+it('should return the largest value in the array', () => {
+    expect(funcs.largestNumber([1, 2, 3, 4, 5])).toBe(5);
+    expect(funcs.largestNumber([10, 20, 30, 40, 50])).toBe(50);
+    expect(funcs.largestNumber([111, 222, 333, 444, 555])).toBe(555);
+})
```
Comment (adfaris) [inline basic-javascript/project-2.spec.js]: I would suggest that you do at least one edge case.
